src/ui/OmniboxResultList/OmniboxResultList.ts

```diff
@@ -289,11 +301,15 @@ export class OmniboxResultList extends ResultList implements IComponentBindings
     });
   }
 
-  private handleQueryOverride(args: IBuildingQueryEventArgs) {
+  protected handleBuildingQuery(args: IBuildingQueryEventArgs) {
     Assert.exists(args);
+    super.handleBuildingQuery(args);
```

Comment:
🤔 ?

I think the way the code is right now, it's going to duplicate the work/ query expression...